cardano-testnet/test/Spec/Plutus/Script/TxInLockingPlutus.hs

```diff
@@ -47,28 +48,29 @@ hprop_plutus = H.integration . H.runFinallies . H.workspace "chairman" $ \tempAb
 
   path <- H.evalIO $ fromMaybe "" <$> IO.lookupEnv "PATH"
 
-  let execConfig = H.ExecConfig
-        { H.execConfigEnv = Last $ Just
-          [ ("CARDANO_CLI", cardanoCli)
-          , ("BASE", projectBase)
-          , ("WORK", tempAbsPath)
-          , ("UTXO_VKEY", tempAbsPath </> "shelley/utxo-keys/utxo1.vkey")
-          , ("UTXO_SKEY", tempAbsPath </> "shelley/utxo-keys/utxo1.skey")
-          , ("CARDANO_NODE_SOCKET_PATH", IO.sprocketArgumentName (head bftSprockets))
-          , ("TESTNET_MAGIC", show @Int testnetMagic)
-          , ("PATH", path)
-          , ("RESULT_FILE", resultFile)
-          ]
-        , H.execConfigCwd = Last $ Just tempBaseAbsPath
-        }
+  unless (OS.isWin32) $ do
```

Comment:
I think it would be more obvious to use `ignoreOnWindows` in `cardano-testnet/test/Main.hs` instead rather than having `OS.isWin32` buried here.

Reply:
Fixed.  Thanks!